usecase/parser/msg.go

```diff
@@ -127,7 +127,9 @@ func ParseBlockTxsMsgToCommands(
 				"/cosmos.vesting.v1beta1.MsgCreateVestingAccount",
 
 				// ethermint evm
-				"/ethermint.evm.v1.MsgEthereumTx":
+				"/ethermint.evm.v1.MsgEthereumTx",
+				"/ethermint.types.v1.LegacyTx",
```

Comment:
Is this a inner type or a message type?

Reply:
Oh, yes. it is inner type. I will remove it from here.